### DISCUSSION TITLE
Error saat stepwise regression: backward elimination

### DISCUSSION THREAD

**Discussion by ChristoNindyo** [teamalgoritma/community#269] [Q&A] [answered]
Salam, bapak dan ibu.

Saya ingin bertanya mengenai machine learning dengan stepwise regression: backward elimination. Berikut saya lampirkan : 
![Error Linear Regression](https://user-images.githubusercontent.com/105972495/184300474-c02fc001-ae16-4fa8-9395-3cc1c2216295.JPG)
![Modeling Linear Regression](https://user-images.githubusercontent.com/105972495/184300645-ce9223b4-bdd1-48fe-94b5-36329df052cf.JPG)

**Comment by idoali** [comment, marked as answer]
Siang pak

Bisa diselesaikan dengan `stats::step(PremiumPrice ~ ., data=data_train)`. Masalah ini muncul karena ada beberapa library yang memiliki nama fungsi yang sama pak. Solusinya adalah menuliskan nama library di bagian depan sebelum nama fungsi.

**Comment by ChristoNindyo** [comment]
Terima kasih, mas
